README.md

```diff
@@ -47,8 +47,8 @@ The frontend [proxies requests](https://create-react-app.dev/docs/proxying-api-r
 
 Api documentation uses [Redoc](https://github.com/Redocly/redoc) to serve documentation files. These files can be found in the `docs/openapi` folder. Api documentation should be split into separate files when appropriate to prevent huge hard to grasp yaml files.
 
-We use an AWS Opensearch docker image (Elasticsearch fork) and require that the following variables get added to the env file.
-* `AWS_ELASTICSEARCH_ENDPOINT=http://localhost:9200`
+We use an AWS OpenSearch docker image (Elasticsearch fork) and require that the following variables get added to the env file.
+* `AWS_ELASTICSEARCH_ENDPOINT=http://opensearch-node1:9200`
```

Comment:
not to nitpick, but we should probably say like "if you are developing locally, use http://localhost:9200 and if you are using docker use http://opensearch-node1:9200", because I think that is the case

Reply:
There are two sections one for local one for docker, Its under the docker section.